handlers/common/common.go

```diff
@@ -4,23 +4,25 @@ import (
 	"context"
```

Comment:
common.go is "common to providers" and was created when all of the individual provider packages were created (adfs.go, oidc.go).  all of these `providers` are going to move soon, probably to `/pkg/providers/*`

`FindJWT` and `ClaimsFromJWT` do need a better home but I don't think this is it.  Perhaps `jwtmanager` would be better.

Reply:
Done